app/models/inventory_dashboard.rb

```diff
@@ -0,0 +1,16 @@
+class InventoryDashboard < ActiveRecord::Base
```

Comment:
class definition in method body
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)